containers/python-3-anaconda-postgres/test-project/hello.py

```diff
@@ -0,0 +1,54 @@
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
+
+#%%
+import os
+import matplotlib
+import matplotlib.pyplot as plt
+import numpy as np
+import psycopg2
+
+try:
+    pg_user = os.getenv('POSTGRES_USER')
```

Comment:
Should `assert(pg_user != None)` be added and for the other env too?

Reply:
Changed this to fail gracefully if the environment variable is not set